### PR TITLE
SED-1408 adapted row height and font size

### PR DESCRIPTION
**Pull request by chillbirdo**
image

### CODE REVIEW

**Review by neogucky — CHANGES_REQUESTED** [review]
Here we are changing the design for all tables to solve an edgecase. Let's just hide the lable for now until we can find a solution with our designer.

**Comment by chillbirdo** [comment]
considered @dvladir comment in Jira

File: projects/step-core/src/lib/modules/table/components/table/table.component.scss [hunk 1]
```diff
@@ -46,6 +46,10 @@
     border-radius: 0 0 16px 16px;
   }
 
+  tr[mat-header-row] {
+    height: 63px;
+  }
```
Comment (neogucky) [inline projects/step-core/src/lib/modules/table/components/table/table.component.scss:51]: This is still the incorrect row height

**Review by neogucky — CHANGES_REQUESTED** [review]
We still have to adjust the row height. Also I would prefer to use styles in legacy-styles to hide the error because:
- I don't like commented out code checked in
- We can't forget about these styles when we start working on 3.21 (since we will delete the legacy-styles first thing)

**Pull request closed without merging**